### DISCUSSION TITLE
Arrow Operators (-> and ->>)

### DISCUSSION THREAD

**Discussion by tinygiant98** [sqlitebrowser/sqlitebrowser#3794] [Q&A]
I've as yet been unable to use the arrow operators (`->` and `->>`) as defined in sqlite's json1 supplement (added in 3.38.0) without an error.  I can use them when querying json data through other means, just not through db browser for sqlite (execute sql tab).  Is this a known limitation or am I attempting to use it incorrectly?

Example, if `t.value` is valid json, shouldn't `t.value -> '$.path'` work?  If not, how do I change the call to make it work?

**Comment by mgrojo** [comment]
Does that work under the sqlite3 cli?

**Reply by tinygiant98** [reply]
I never use the cli, mostly through another program as I used embedded sqlite.  The arrow operators work in all of the queries sent through those means, but never work in this gui program.
